docker-compose.yml

```diff
@@ -16,14 +16,14 @@ services:
        - SKIP_FORCE_SSL=true
        - WEB_URL=postgis://postgres:secret@postgres/school_experience
      depends_on:
-       - create-db
+       - db-tasks
 
    delayed-jobs:
      image: school-experience:latest
      command: rake jobs:work
      restart: always
      healthcheck:
-       disable: true
```

Comment:
This should probably stay as true. 

The health check will fail, because it tests for the presence of an http endpoint which isn't available when running dj instead of puma

Reply:
I'll see whether it changes the behaviour at all.

Reply:
yep no problems